lib/scrapers/redditscrape.py

```diff
@@ -0,0 +1,105 @@
+import praw
+from tornado import gen
+from lib.config import CONFIG
+
+
+class RedditScraper(object):
+    name = 'reddit'
+
+    @property
+    def num_scrape(self):
+        if CONFIG['_mode'] == 'prod':
+            return 1000
+        return 100
+
+    @gen.coroutine
+    def scrape(self, user_data):
+        try:
+            tokens = user_data.services['reddit']
+        except KeyError:
+            return False
+        if 'denied' in tokens:
+            return False
+        client_id = CONFIG.get("reddit_client_id"),
+        client_secret = CONFIG.get("reddit_client_secret")
+
+        r = praw.Reddit(user_agent="QS-agent/1.0")
+        r.set_oauth_app_info(client_id=client_id, client_secret=client_secret, redirect_uri="https://iamadatapoint.com/auth/reddit")
+        r.set_access_credentials(**tokens)
```

Comment:
@mynameisfiber were you able to get the reddit scraper to create it's auth object correctly in the end? I saw you made some changes to the file.